src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -7,7 +7,7 @@
     <IsPackable>false</IsPackable>
     <VersionPrefix>4.5.1</VersionPrefix>
     <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
-    <AssemblyVersion>4.0.3.0</AssemblyVersion>
+    <AssemblyVersion>4.0.3.0</AssemblyVersion> <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
```

Comment:
I saw that you wrote in [the issue](https://github.com/dotnet/maintenance-packages/issues/16):

> I would just move the fixed assembly version somewhere else so that whoever increments the versions

If we really want to keep this simple for developers when they need to enable the package for building, then we should consider putting the properties that they should not modify in a different location/file. What do you think?

Reply:
I'm not really a fan of different property groups in msbuild (unless when applying conditions to those) because they are verbose. When I wrote that comment I then later realized that instead of a positional change, a comment marker would be better so that it works with a bot as well. I kind of like that version properties are grouped together so that people can just ignore them altogether.

Reply:
Okay. As long as we have the process clearly documented, people don't need to mess with anything other than what we tell them to modify.